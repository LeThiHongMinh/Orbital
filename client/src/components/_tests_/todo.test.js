import * as React from 'react'
import ToDoList from '../Todo'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<ToDoList />)
  expect(tree).toMatchSnapshot()
})