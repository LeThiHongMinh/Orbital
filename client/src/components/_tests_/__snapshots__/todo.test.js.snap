// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="fixed bottom-8 right-8 p-4 bg-red-100 shadow-lg rounded-lg w-full max-w-md"
>
  <h3
    className="text-2xl font-semibold mb-4 text-red-700"
  >
    To-Do List
  </h3>
  <form
    className="flex mb-4"
    onSubmit={[Function]}
  >
    <input
      className="border border-red-300 p-2 rounded-l-lg flex-grow focus:outline-none focus:border-red-500"
      onChange={[Function]}
      placeholder="Enter a new task"
      type="text"
      value=""
    />
    <button
      className="bg-red-500 text-white p-2 rounded-r-lg hover:bg-red-600 transition"
      type="submit"
    >
      Add
    </button>
  </form>
  <ul
    className="list-disc list-inside"
  />
</div>
`;
